odl/discr/discr_ops.py

```diff
@@ -126,6 +127,10 @@ def finite_diff(f, out=None, axis=0, dx=1.0, edge_order=2,
     else:
         dx = float(dx)
 
+    method = str(method).lower()
+    if method not in ('central', 'forward', 'backward'):
+        raise ValueError('method {} has to be central, forward or backward')
```

Comment:
This will print the curly brackets literally. Print the original argument, not the sanitized one. And use quotes in the string since the possible values are strings.